opencti-platform/opencti-front/lang/front/en.json

```
@@ -3058,6 +3067,7 @@
   "The password has been updated": "The password has been updated",
   "The public dashboard is disabled": "The public dashboard is disabled",
   "The relations attached to selected entities will be copied to the merged entity.": "The relations attached to selected entities will be copied to the merged entity.",
+  "The reset code you entered is invalid or has expired. You can request a new code, but please wait 30 seconds after your last request before trying again.": "The reset code you entered is invalid or has expired. You can request a new code, but please wait 30 seconds after your last request before trying again.",
```

Feel free to refuse this one, it's really a feeling :)
```suggestion
  "The reset code you entered is invalid or has expired. You can request a new code, but please wait 30 seconds after your last request before trying again.": "The reset code you entered is invalid or has expired. You can request a new code after a delay of 30 seconds.",
```